index.ts

```diff
@@ -29,7 +29,7 @@ function fizzBuzz(n: number) {
             strings.splice(j, 0, "Fezz");
         }
 
-        if(i % 17 == 0)
+        if(i % 17 == 0 && part2_flag)
             strings.reverse();
```

Comment:
part1_flag and part2_flag, you might want to name them around their functionality purpose, rather than the requirement number